en/docs/ai/deploy-an-ai-agent-created-using-wso2-ballerina-integrator.md

```diff
@@ -0,0 +1,83 @@
+# Deploy an AI Agent Created Using WSO2 Ballerina Integrator
+
+This guide walks you through deploying an AI Agent built with WSO2 Ballerina Integrator to Devant, using the **Deploy to Devant** option available in the Ballerina Integrator UI.
+
+!!! note
+        This guide assumes you have already created an AI Agent by following the tutorial [Integrating AI Agents with External Endpoints](https://bi.docs.wso2.com/integration-guides/ai/agents/integrating-agents-with-external-endpoints/).
+
+## Pre-requisites
+
+Before you begin, ensure that you have:
+
+* Installed **WSO2 Ballerina Integrator** on your local machine.
+* Created an AI Agent using the [AI Agent Integration Tutorial](https://bi.docs.wso2.com/integration-guides/ai/agents/integrating-agents-with-external-endpoints/).
+* A **Devant account** with access to at least one **organization** and **project**.
+* A **GitHub account** with a public repository to push your integration code.
+
+## Step 1: Start the Deployment from Ballerina Integrator
```

Comment:
All headings except L1 headings should be in sentence case

Reply:
Done.

Reply:
<img width="3168" height="7890" alt="image" src="https://github.com/user-attachments/assets/cd49d3e5-d180-4c0f-8534-91ba9aacabfd" />

Reply:
Shouldn't we use the name `BI` or `WSO Integrator BI` as the name.

Reply:
@sm1990 let's use `WSO2 Integrator: BI`